internal/test/testdata/store/resource_policies/policy_04.yaml

```diff
@@ -5,7 +5,7 @@ resourcePolicy:
   version: "default"
   importDerivedRoles:
     - apatr_common_roles
-  resource: "album:object"
+  resource: "album_object"
```

Comment:
Why do we need to change it? `album:object` is valid, isn't it?

Reply:
Careless usage of `sed` 😄